docs/gallery-submission.md

```diff
@@ -131,6 +131,14 @@ Installation instructions have two parts:
   * Installing the driver
 - Your custom installation instructions
 
+**Note on Prep Builder Instructions:** If your connector can be used with Prep Builder, add information about where to put the TACO file.    
```

Comment:
Seems like this content needs to end up in block below.

Also probably needs to be reflected [here](https://tableau.github.io/connector-plugin-sdk/docs/run-taco) as well